.github/workflows/debug-environments.yml

```
@@ -0,0 +1,33 @@
+name: Changing Environments
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  build:
+    name: "🔨 Build in ${{ matrix.environment }}"
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    environment: ${{ matrix.environment }}
+    strategy:
+      matrix:
+        environment:
+          - ""
+          - "Never debugging environment"
+          - "Always debugging environment"
+    steps:
+      - name: "environment: {{ jobs.validate.environment }}"
```

```suggestion
      - name: "environment: ${{ jobs.validate.environment }}"
```
can't seem to reference my own env...